src/components/CookieNotice/CookiePolicy.scss

```diff
@@ -0,0 +1,50 @@
+@import "constants/style.scss";
+
+$cookie-consent-button-text-size: 18px;
+
+.portal {
+  border-radius: 5px;
+}
+
+.cookie-policy {
+  margin: $margin--small;
+}
+
+.cookie-policy__title {
+  display: block;
```

Comment:
... you should not set `display: block` explicitly if it is the default of the HTML elements. It just bloats the file size of the CSS and will not change the look & feel